osbenchmark/metrics.py

```diff
@@ -1706,6 +1706,7 @@ def __call__(self):
                         task.operation.name,
                         self.summary_stats("throughput", t, op_type),
                         self.single_latency(t, op_type),
+                        self.single_latency(t, op_type, metric_name="client_processing_time"),
```

Comment:
Does it make sense to call this `client_time`? Or does this not include client waiting?

Reply:
Here, client_processing_time is calculated as (total request time - service time). Based on my understanding, it does not include client waiting.